nearlib/test/integration.test.js

```diff
@@ -43,6 +49,37 @@ test('create account and then view account returns the created account', async (
     fail('exceeded number of retries for viewing account');
 });
 
+test('deploy contract and make function calls', async () => {
+    var data = [...fs.readFileSync('../tests/hello.wasm')];  
+    const initialAccount = await account.viewAccount(aliceAccountName);
+    var deployResult = await nearjs.deployContract(
+        aliceAccountName,
+        "test_contract",
+        data,
+        "FTEov54o3JFxgnrouLNo2uferbvkU7fHDJvt7ohJNpZY");
+    waitForNonceToIncrease(initialAccount);
+    const args = {
+        "name": "trex"
+    };
+    var viewFunctionResult = await nearjs.callViewFunction(
+        aliceAccountName,
+        "test_contract",
+        "near_func_hello", // this is the function defined in hello.wasm file that we are calling
+        args);
+    expect(viewFunctionResult).toEqual("hello trex");
+});
+
+const waitForNonceToIncrease = async (initialAccount) => {
+    for (i = 0; i < TEST_MAX_RETRIES; i++) {
+        try {
+            const viewAccountResponse = await account.viewAccount(initialAccount['account_id']);
+            //console.log(viewAccountResponse);
+            if (viewAccountResponse['nonce'] != initialAccount['nonce']) { return; }
+        } catch (_) {}
```

Comment:
why is swallowing errors needed? It's best to explain it in `catch` block

Reply:
The errors that we get right now are currently extremely long and unreadable. I am currently doing a lot of eyeballing of what requests are going out and succeeding/failing, so not having spammy logs is a good thing at the moment. Error handling is probably worth revisiting and doing a full sweep once the basics are there (and once some real client is using this lib).

Reply:
Just have a comment then, something like

```
catch (_) {
   // Errors are expected, not logging to avoid spam
}
```